packages/hoppscotch-app/src/components/profile/Picture.vue

```diff
@@ -65,6 +71,10 @@ export default defineComponent({
   },
   methods: {
     toHex(initial: string) {
+      if (!this.isValidInitial(initial)) {
+        initial = "HS"
```

Comment:
Parameter Reassign is generally frowned upon. As based on the values passed around, it can bring upon call by reference or call by value rules. Just isolate this to a different variable. Variables are basically free.

I will add an ESLint rule (`no-param-reassign`) to prevent this in the future.